src/int.js

```diff
@@ -142,10 +142,8 @@ Sk.builtin.int_ = Sk.abstr.buildNativeClass("int", {
         }, JSBI.leftShift),
         nb$rshift: numberShiftSlot(
             (v, w) => {
-                const tmp = v >> w;
-                if (v > 0 && tmp < 0) {
-                    return tmp & (Math.pow(2, 32 - w) - 1);
-                }
+                // Avoid forced signed 32 bit conversion and just divide.
+                const tmp = Math.floor(v / shiftconsts[w+1]);;
                 return tmp;
```

Comment:
Suggestion - can we tidy this up - remove the extra semi colon and the `tmp` variable.

Reply:
Thank you for the quick code review.

I completely missed that extra semi colon.